### ISSUE TITLE
Notifications don't go away

### ISSUE THREAD

**Issue by chrismore** [mozilla/all-aboard#76]
I used this config:

`{
    "defaultSidebarInterval": 5,
    "timeElapsedFormula": 1000,
    "waitInterval": 5000
}
`

I got a notification every 5 seconds regardless if I engaged with the sidebar or not and even after getting to the final sidebar, I kept getting notifications. They never ended.

**Comment by schalkneethling** [comment]
@chrismore see if this is resolved once https://github.com/mozilla/all-aboard/pull/81 is merged.

**Comment by schalkneethling** [comment]
@chrismore Can you confirm that this is resolved? Thanks!

**Comment by chrismore** [comment]
@schalkneethling fixed! But there is a new issue, clicking the chrome icon doesn't close the sidebar. It just refreshes it. I can't close it unless I click the [X].

**Comment by schalkneethling** [comment]
@chrismore yup, this is the next thing to do. I need to see the best way to handle that. I have an idea though. Will open and issue for that.
